src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,11 @@
-export const UserInfo = () => (
-  <>Put the user here</>
-);
+export const UserInfo = ({ user }) => {
```

Comment:
According previous comment user will be exist, so you can destructure it here

Reply:
Not fixed